### ISSUE TITLE
create-dmg is out-of-date

### ISSUE THREAD

**Issue by Abhinavreddy-B** [Neuri-AI/PPG#9]
The create-dmg source code being used is out of date, there was this one particular bug with bliss command on silicon devices which breaks the create-dmg command. this bug has been resolved by create-dmg team in their new release, so please update the create-dmg code with the latest one. Shall i create a PR for that?

**Comment by runesc** [comment]
Yes, you should definitely create the pull request! It seems like an important fix and I'm sure users of silicon devices will appreciate it.

**Comment by runesc** [comment]
Hi @Abhinavreddy-B  this issue has been fixed in commit https://github.com/Neuri-AI/PPG/commit/a5c05c5605ed4abcb331fe5070907d6fa733c39b, and the fix is now available in version 1.2.0 of the project.
